ministryofjustice/analytics-platform-control-panel
# Feature/run black

## :memo: Summary
This PR contributes to https://github.com/ministryofjustice/data-platform/issues/3374

Specifically it:
- Runs `black` on the whole codebase to bring a consistent style across all files. 
- Add super-linter CI job. We are using this to primarily run black, flake8 and isort, however we may utilise some of the other linters in the future

## :technologist: How should the reviewer test these changes?
- Pull the changes, update your dev dependencies
- Run ` black . && flake8 . && isort .` from your terminal to run all three checks. There should be no proposed changes
- All checks in CI are passing

## :books: Documentation status
